package.json

```diff
@@ -31,13 +30,6 @@
     "eslint-plugin-flowtype": "^3.11.1"
   },
   "peerDependencies": {
-    "babel-eslint": "^10.0.1",
-    "eslint": "^5.9.0",
-    "eslint-plugin-cypress": "^2.2.1",
-    "eslint-plugin-mocha": "^5.3.0",
-    "eslint-plugin-spellcheck": "0.0.11",
-    "eslint-config-airbnb-base": "^13.2.0",
-    "eslint-plugin-import": "^2.18.0",
-    "eslint-plugin-flowtype": "^3.11.1"
```

Comment:
Why is all of this removed? Are these outdated peer dependencies?

Reply:
They are in the `dependencies` object, they shouldn't be `peerDependencies`